include/operations/ClassicControlledOperation.hpp

```diff
@@ -124,10 +138,23 @@ class ClassicControlledOperation final : public Operation {
                     const RegisterNames& creg) const override {
     of << "if(";
     of << creg[controlRegister.first].first;
-    of << " == " << expectedValue << ") ";
+    of << " " << comparisonKind << " " << expectedValue << ") ";
     op->dumpOpenQASM(of, qreg, creg);
   }
 
+  void dumpOpenQASM3(std::ostream& of, const RegisterNames& qreg,
+                     const RegisterNames& creg,
+                     uint32_t indent = 0) const override {
```

Comment:
```suggestion
                     uint32_t indent) const override {
```
no default arguments allowed unfortunately. Why's that indent necessary anyways? Was this just poorly handled in the OQ2 export?
Same warning holds for all other overrides of this method.